output/cloud/expv2/hdr.go

```diff
@@ -159,7 +159,7 @@ func histogramAsProto(h *histogram, time int64) *pbcloud.TrendHdrValue {
 	}
 	// We don't expect to change the minimum resolution at runtime
 	// so it is safe use directly a pointer without creating a copy
-	hval.MinResolution = &h.MinimumResolution
+	hval.MinResolution = h.MinimumResolution
```

Comment:
:heart: but please also clean up the comment which became obsolete

Reply:
Done